### PR TITLE
Add column and get data

### PR DESCRIPTION
**Pull request by zkr024**
* Add species column
* Insert more data into the table
* Set transaction to update the values in the species column
* Use conditions to set species between Pokemon and Digimon
* Use setpoints and rollback functions.
* Use queries to get specific data from the table.

### CODE REVIEW

File: data.sql [hunk 1]
```diff
@@ -1,14 +1,64 @@
...
+	name, date_of_birth, escape_attempts, neutered, weight_kg)
+	VALUES ('Squirtle', '1993-04-02', 3, false, -12.13);
+INSERT INTO public.animals(
+	name, date_of_birth, escape_attempts, neutered, weight_kg)
+	VALUES ('Angemon', '2005-06-12', 1, true, -45);
+INSERT INTO public.animals(
+	name, date_of_birth, escape_attempts, neutered, weight_kg)
+	VALUES ('Boarmon', '2005-06-07', 7, true, 20.4);
+INSERT INTO public.animals(
+	name, date_of_birth, escape_attempts, neutered, weight_kg)
```
Comment (sja-thedude) [inline data.sql]: - [ ] Kindly place all the values in `' '` inverted commas. Example-: 
```
INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
    VALUES ('Plantmon', '2022/11/15', '2', 'true', '-5.7');
```
Make sure to make changes for all animals.

File: queries.sql [hunk 2]
```diff
@@ -8,3 +8,22 @@ SELECT name, escape_attempts from animals where weight_kg > 10.5;
...
+
+-- How many didn't try to escape
+select count(*) from animals where escape_attempts = 0;
+
+-- Average weight
+select avg(weight_kg) from animals;
+
+-- Who escapes the most
+select max(escape_attempts) from animals;
+select name from animals where escape_attempts = 7;
```
Comment (sja-thedude) [inline queries.sql]: - [ ] Kindly make sure to add the query as per the question. The question is `Who escapes the most, neutered or not neutered animals?` You have not added a query for neutered along with escapes the most. Kindly do that.

File: data.sql [hunk 3]
```diff
@@ -1,14 +1,64 @@
...
+select * from animals;
+
+begin;
+delete from animals where date_of_birth > '2022-01-01';
+select * from animals;
+SAVEPOINT deleted;
+update animals set weight_kg = weight_kg * - 1;
+select * from animals;
+update animals set weight_kg = weight_kg * - 1 where weight_kg < 0;
+select * from animals;
```
Comment (sja-thedude) [inline data.sql]: - [ ] Great job in writing the transactions correctly, however you have to place line number 37 to 64 in queries.sql before the questions part.

**Review by sja-thedude — CHANGES_REQUESTED** [review]
## Required Changes ♻️

Hi @zkr024, This is @sja-thedude your code reviewer.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

768070f3-bc9a-4aba-b8bc-25793e00ea1e_text

## Highlights
✔️ Good job so far
✔️ Professional Readme
✔️ Correct Gitflow

## Required 
- [ ] _Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by SajjadAhmad14 — APPROVED** [review]
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:

### Hi Oscar, 
## Congratulations 🎉 Thank you for implementing all the requirements 🙏
### Please go ahead and merge the branch!

## Happy Coding 👏👏👏
#### Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
#### Please remember to tag me in your question so I can receive the notification.
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**